packages/grafana-ui/src/components/InteractiveTable/InteractiveTable.story.tsx

```diff
@@ -1,9 +1,9 @@
 import { Meta, StoryFn } from '@storybook/react';
-import React, { useMemo } from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 
 import { InteractiveTable, Column, CellProps, LinkButton } from '@grafana/ui';
 
-import { InteractiveTableHeaderTooltip } from './InteractiveTable';
+import { FetchDataArgs, InteractiveTableHeaderTooltip } from './InteractiveTable';
 import mdx from './InteractiveTable.mdx';
 
 const EXCLUDED_PROPS = ['className', 'renderExpandedRow', 'getRowId'];
```

Comment:
We should add `fetchData` to the `EXCLUDED_PROPS` here since we can't configure it.